fatiando/gravmag/transform.py

```diff
@@ -262,6 +263,53 @@ def tga(x, y, data, shape, method='fd'):
     return res
 
 
+def tilt(x, y, data, shape):
+    """
+    Calculates the potential field tilt, as defined by Miller and Singh (1994):
+
+    .. math::
+        tilt(f) = tan^{-1}(\\frac{\\frac{dT}{dz}}{\\sqrt{\\frac{dT}{dx}^2 +
+        \\frac{dT}{dy}^2}})
+
+    .. note:: Requires gridded data.
+
+    .. warning::
+
+        If the data is not in SI units, the derivatives will be in
+        strange units and so will the potential field tilt! I strongly
+        recommend converting the data to SI **before** calculating the
+        TGA is you need the gradient in Eotvos (use one of the unit conversion
+        functions of :mod:`fatiando.utils`).
```

Comment:
Is this really a problem? The tilt is dimensionless so the units the data are in shouldn't matter much, so long as they are compatible.